src/Accordion/Accordion.tsx

```diff
@@ -5,9 +5,7 @@ type AccordionProps = React.HTMLAttributes<HTMLDivElement> & {
 };
 
 const Accordion = ({ accordion, ...rest }: AccordionProps): JSX.Element => {
-    const role = accordion ? 'tablist' : undefined;
-
-    return <div role={role} {...rest} />;
+    return <div {...rest} />;
```

Comment:
Seems a bit redundant (even though I personally would wrap it in that div most of the time). As long as each accordion item has unique id's that aren't tied to a top level accordion grouping could be ok to remove. 

It's not essential to remove now though right? Flag that in the next release, should just aim to fix this incorrect role assignment for this PR maybe?

Reply:
> even though I personally would wrap it in that div most of the time

Yup, me too. Seems we agree though - that if it's not necessary to be spec-compliant then it doesn't belong in this package?

> Not sure I fully understand what you mean by backwards incompatible

Users may be relying on the current `AccordionItem` rendering a `div`, and upgrading would cause those users' apps to break. Therefore it would be a backward-incompatible change. This is not really a concern though, as the next release is going to be a major release, and users should therefore expect some backward-incompatible changes.

> It's not essential to remove now though right? Flag that in the next release, should just aim to fix this incorrect role assignment for this PR maybe?

That's fine for this PR, yes - I'll create an issue to flag this concern separately. Note that it won't be possible to create an independent release for this patch though, as `master` already has some breaking changes in it. It will be bundled as part of the `3.X` released planned for the near future.